### PR TITLE
update spec to 1.1.0 release

### PR DESCRIPTION
**Pull request by surajssd**
Update to new version 1.1.0 https://github.com/kubernetes/kompose/releases/tag/v1.1.0

Second commit also fixes https://bugzilla.redhat.com/show_bug.cgi?id=1478152

### CODE REVIEW

File: kompose.spec [hunk 1]
```diff
@@ -52,26 +52,21 @@
...
+Version:        1.1.0
+Release:        0.1%{?dist}
 Summary:        Tool to move from 'docker-compose' to Kubernetes
 License:        ASL 2.0
 URL:            https://%{provider_prefix}
 Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 
 # e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
-ExclusiveArch:  %{ix86} x86_64 %{arm} aarch64 s390x
+ExclusiveArch:  %{?go_arches:%{go_arches}}%{!?go_arches:%{ix86} x86_64 aarch64 %{arm}}
```
Comment (surajssd) [inline kompose.spec]: @praveenkumar should I keep it as it was before ?

Comment (surajssd) [reply kompose.spec]: This is auto generated by gofed

Comment (praveenkumar) [reply kompose.spec]: @surajssd yes please keep it as it was before so fedora builds works as expected.

Comment (surajssd) [reply kompose.spec]: done

**Pull request merged**